config/application.rb

```diff
@@ -86,5 +86,7 @@ class Application < Rails::Application
       min_threads: 1,
       max_threads: 3,
       idletime:    30.seconds
+
+    config.cache_store = :mem_cache_store, '127.0.0.1:11211', { pool: { size: 10 } }
```

Comment:
Put the cache store ip/port into config. Allow per env setting.

Reply:
If we ran it on multiple machines then sure. But since today in dev and prod we run on localhost, less duplication is better.

Reply:
OK. Makes sense. We can configurate when needed.